azure-pipelines.sds.pr-release.yml

```diff
@@ -0,0 +1,489 @@
+name: $(Date:yyyyMMddhhmm)-$(Rev:r)
+
+resources:
+  repositories:
+    - repository: azTemplates
+      type: github
+      name: hmcts/azure-devops-templates
+      ref: master
+      endpoint: hmcts
+      
+trigger: none
+
+pr:
+  - master
+
+variables:
+  - group: vh-github-app-credentials
+  - group: vh-admin-web
+  - name: run_acceptance_tests
+    value: false
+
+stages:
+#####################################################
+# CI Build Checks. ##################################
+  - stage: CI_Build
+    displayName: CI Build
+    variables: 
+      - template: variables/shared.yaml
+        parameters:
+          env: ''
+    jobs:
+      - job: CI_Helm
+        displayName: "Helm CI"
+        steps:
+          - checkout: self
+
+          - template: templates\Containerisation\helm\helm-lint.yaml@azTemplates
+            parameters:
+              chartPath: ${{ variables.chartPath }}
+              chartValuesPath: ${{ variables.chartPath }}
+
+      # - job: UnitTests
```

Comment:
does this project not have unit tests?

Reply:
yeah, they seem to be working now... haha